src/mono/browser/browser.proj

```diff
@@ -576,8 +576,6 @@
       <_MonoRollupEnvironmentVariable Include="WasmEnableThreads:$(WasmEnableThreads)" />
       <_MonoRollupEnvironmentVariable Include="WASM_ENABLE_SIMD:1" Condition="'$(WasmEnableSIMD)' != 'false'" />
       <_MonoRollupEnvironmentVariable Include="WASM_ENABLE_SIMD:0" Condition="'$(WasmEnableSIMD)' == 'false'" />
-      <_MonoRollupEnvironmentVariable Include="WASM_PERFTRACING:1" Condition="'$(WasmPerfTracing)' == 'true'" />
```

Comment:
The variable is used in other places as well https://github.com/search?q=repo%3Adotnet%2Fruntime%20WASM_PERFTRACING&type=code, are those still relevant?

Reply:
yes, the idea of this change is that we are don't rollup different JS depending on this.
In other places, it's about re-link.